lib/hashes.md

```diff
@@ -5,40 +5,53 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a Hash?
 
-Replace me with your response...
+A dictionary like collection of data that pairs unique keys to their corresponding vaules.
 
 # What are some examples of information that would be Hashes as opposed to some other data type?
 
-Replace me with your response...
+Keys can be any data type.
+Strings and numbers would be the two most common data types. 
+
 
 # How are Hashes and Arrays similar? How are they different?
 
-Replace me with your response...
+Hashes are special arrays.  
```

Comment:
I'd like to caution against thinking of Hashes as kinds of Arrays. Both are *collections*, however.